rs/tests/message_routing/xnet/xnet_slo_120_subnets_staging_test.rs

```diff
@@ -16,7 +16,7 @@ const PER_TASK_TIMEOUT: Duration = Duration::from_secs(800);
 const OVERALL_TIMEOUT: Duration = Duration::from_secs(1400);
 
 fn main() -> Result<()> {
-    let config = Config::new(SUBNETS, NODES_PER_SUBNET, RUNTIME, REQUEST_RATE);
+    let config = Config::new(SUBNETS, NODES_PER_SUBNET, RUNTIME, REQUEST_RATE).with_prometheus();
```

Comment:
Is it maybe too expensive to always deploy a prometheus instance, i.e., also in CI tests? So far we typically only added the `with_prometheus()` on demand when running locally...